.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,92 @@
+
+        * General Dash questions and/or support requests are best directed to the [Dashpay Reddit](https://www.reddit.com/r/dashpay/).
+        * To report critical security issues, email infosec@dash.org and/or see the DASH bug bounty program on [Bugcrowd](https://bugcrowd.com/dashdigitalcash) instead of creating Github issues.
+        * If the node is "stuck" during sync or giving "block checksum mismatch" errors, please ensure your hardware is stable by running `memtest` and observe CPU temperature with a load-test tool such as `linpack` before creating an issue.
+
+        ----
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)